mission-control/docs/config-db/concepts/retention.md

```diff
@@ -4,11 +4,11 @@ After scraping we can choose to retain results on the basis of name, age, count
 
 The retention rules are applied for each unique catalog item. If `changes` is specified with type `X` and count `20`, last 20 changes of `X` type would be kept for each catalog item
 
-| Field     | Description                         | Scheme                                    | Required |
-| --------- | ----------------------------------- | ----------------------------------------- | -------- |
-| `changes` | Specify retention rules for changes | [`[]RetentionChanges`](#retentionchanges) | `false`  |
+| Field     | Description                         | Scheme                                   | Required |
+| --------- | ----------------------------------- | ---------------------------------------- | -------- |
+| `changes` | Specify retention rules for changes | [`[]ChangeRetention`](#change-retention) | `false`  |
 
-## RetentionChanges
+## Change Retention
```

Comment:
Maybe this should move to changes.md and also add a section there around excluding and transforming changes